contracts/DefaultStrategy.sol

```diff
@@ -0,0 +1,517 @@
+        uint256 availableVotes;
+        (sortedGroups, availableVotes) = getSortedGroupsWithVotes(votableGroups);
+        availableVotes += votes;
+
+        uint256[] memory votesPerGroup = new uint256[](votableGroups.length);
+        uint256 groupsVoted = votableGroups.length;
+        uint256 targetVotes = availableVotes / groupsVoted;
+
+        /*
+         * This would normally be (i = votableGroups.length - 1; i >=0; i--),
+         * but we can't i-- on the last iteration when i=0, since i is an
+         * unsigned integer. So we iterate with the loop variable 1 greater than
+         * expected, set index = i-1, and use index inside the loop.
+         */
+        for (uint256 i = votableGroups.length; i > 0; i--) {
+            uint256 index = i - 1;
+            if (sortedGroups[index].votes >= targetVotes) {
+                groupsVoted--;
+                availableVotes -= sortedGroups[index].votes;
+                targetVotes = availableVotes / groupsVoted;
+                votesPerGroup[index] = 0;
+            } else {
+                votesPerGroup[index] = targetVotes - sortedGroups[index].votes;
+
+                if (availableVotes % groupsVoted > index) {
+                    votesPerGroup[index]++;
+                }
+            }
+        }
+
+        finalGroups = new address[](groupsVoted);
+        finalVotes = new uint256[](groupsVoted);
+
+        for (uint256 i = 0; i < groupsVoted; i++) {
+            finalGroups[i] = sortedGroups[i].group;
+            finalVotes[i] = votesPerGroup[i];
+        }
+
+        return (finalGroups, finalVotes);
+    }
+
+    /**
+     * @notice Distributes withdrawals from default strategy by computing the number of votes that
+     * should be withdrawn from each group, then calling out to
+     * `Account.scheduleWithdrawals`.
+     * @param withdrawal The amount of votes to withdraw.
+     * CELO.
+     * @dev The withdrawal distribution strategy is to:
+     * 1. Withdraw as much as possible from any deprecated groups.
+     * 2. If more votes still need to be withdrawn, try and have each validator
+     * group end up receiving the same amount of votes from the system. If a
+     * group already has less votes than the average of the total remaining
+     * votes, it will not be withdrawn from, and instead we'll try to evenly
+     * distribute between the remaining groups.
+     */
+    function distributeWithdrawalsDefaultStrategy(uint256 withdrawal)
+        external
+        onlyManager
+        returns (address[] memory finalGroups, uint256[] memory finalVotes)
+    {
+        address[] memory deprecatedGroupsWithdrawn;
+        uint256[] memory deprecatedWithdrawalsPerGroup;
+        uint256 numberDeprecatedGroupsWithdrawn;
+
+        (
+            deprecatedGroupsWithdrawn,
+            deprecatedWithdrawalsPerGroup,
+            numberDeprecatedGroupsWithdrawn,
+            withdrawal
+        ) = getDeprecatedGroupsWithdrawalDistribution(withdrawal);
```

Comment:
Not necessary for this PR, but wondering if a slight improvement, given the rebalancing logic we now have, would be to remove this logic from the standard withdrawal, instead have a function in here that rebalances all votes away from deprecated groups to the active ones.

Reply:
Totally agreed - I didnt want to touch default strategy more than I have to in this PR since it is already rather big, but it is in my notes as possible improvement